src/_blog/libp2p-IPFSþing-2023-recap.md

```diff
@@ -0,0 +1,137 @@
+        </figcaption>
+    </figure>
+</div>
+
+Marten Seemann (go-libp2p maintainer, Software Engineer at Protocol Labs)
+Marco Munizaga (go-libp2p and zig-libp2p maintainer, Software Engineer at Protocol Labs)
+
+@[youtube](aDHymXQJ4bs)
+
+Don't miss this presentation by Marten and Marco. They start off showing how libp2p is evolving and improving with the addition of the QUIC transport followed by an explanation of simple it is to create a compatible libp2p stack out of a QUIC library, the libp2p TLS extension and some code for doing peer ID encoding and sending multistream headers. To prove how simple it is, Marco showed of his zig-libp2p implementation sending a pig to a go-libp2p node.
+
+### Connecting everthing, everywhere, all at once with libp2p
+
+Max Inden (rust-libp2p maintainer, Software Engineer at Protocol Labs)
+
+@[youtube](4v-iIB0C9_8)
+
+Max's talk about the [Universal Connectivity demonstrator app](https://github.com/libp2p/universal-connectivity) broke the record for the most people involved. @2color / Daniel, @TheDiscordian, @jochasinga / Pan, @achingbrain / Alex, @maschad / Chad, @p-shahi / Prithvi, @marcopolo / Marco, @thomaseizinger / Thomas, and @mxinden / Max all collaborated to make it happen. The work on the Universal Connectivity demonstrator app turned out to be an excellent driver for [interop testing and the need for further work there](https://github.com/libp2p/interop). The live demo worked in showing a go-libp2p server talking to a rust-libp2p server talking to go-libp2p laptop talking to js-libp2p browsers all using a variety of transports including QUIC, WebRTC Direct, and WebTransport. Max also introduced us to the [new libp2p website focused on connectivity](https://connectivity.libp2p.io/).
+
+### libp2p Workshop
+
+Thomas Eizinger (rust-libp2p maintainer, Software Engineer at Protocol Labs)
+
+During the unconference sessions late in the week, Thomas hosted a libp2p workshop where participants went through the process of building a peer-to-peer chat application using the rust-libp2p. The workshop's code is in [this repo](https://github.com/thomaseizinger/libp2p-workshop.git). You too can give it a try by cloning the code with `--mirror` so that you get all of the branches. This workshop is designed to be followed in iterations. Start by checking out the `iteration-1` branch and follow along in the README.md. Once complete, you can check out the `iteration-2` branch and again follow along the additional steps in the READM.md. The repo contains branches for `iteration-1`, `iteration-2`, `iteration-3`, `iteration-4`, `iteration-5`, and `iteration-final`. When done you should have a functioning peer-to-peer chat application built using rust-libp2p.
```

Comment:
```suggestion
During the unconference sessions late in the week, Thomas hosted a libp2p workshop where participants went through the process of building a peer-to-peer chat application using rust-libp2p. The workshop's code is in [this repo](https://github.com/thomaseizinger/libp2p-workshop.git). You too can give it a try by cloning the code with `--mirror` so that you get all of the branches. This workshop is designed to be followed in iterations. Start by checking out the `iteration-1` branch and follow along in the README. Once complete, you can check out the `iteration-2` branch and again follow along the additional steps in the README. The repo contains branches for `iteration-1`, `iteration-2`, `iteration-3`, `iteration-4`, `iteration-5`, and `iteration-final`. When done you should have a functioning peer-to-peer chat application built using rust-libp2p.
```